src/main/java/mate/academy/controllers/IndexController.java

```diff
@@ -3,22 +3,32 @@
 import java.io.IOException;
 import java.time.LocalTime;
 import java.time.format.DateTimeFormatter;
+import java.util.Set;
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import mate.academy.lb.Injector;
 import mate.academy.model.Product;
+import mate.academy.model.Role;
+import mate.academy.model.User;
 import mate.academy.service.ProductService;
+import mate.academy.service.UserService;
 
 public class IndexController extends HttpServlet {
     private static final Injector injector = Injector.getInstance("mate.academy");
     private final ProductService productService =
             (ProductService) injector.getInstance(ProductService.class);
+    private final UserService userService =
+            (UserService) injector.getInstance(UserService.class);
 
     @Override
     public void init() throws ServletException {
         super.init();
+
+        User admin = new User("admin", "admin", "admin");
+        admin.setRoles(Set.of(Role.of("ADMIN")));
+        userService.create(admin);
```

Comment:
let's create a separate InjectDataController for this purpose (and for creating products as well)
we should not place business logic in the init() method